### ISSUE TITLE
Build package down site for repo

### ISSUE THREAD

**Issue by ndphillips** [ndphillips/FFTrees#216]
We need a package down site for this repo!

**Comment by hneth** [comment]
@ndphillips:  I tried adding support for a **pkgdown** workflow that automatically would update a site at <https://ndphillips.github.io/FFTrees/>.  Unfortunately, this does not seem to work yet.  Perhaps the use of **pages** first needs to be enabled in the repository's GitHub settings?

**Comment by ndphillips** [comment]
Yep it needs to be enabled in the repo settings. I just turned it on and I am seeing the website at https://ndphillips.github.io/FFTrees/

Looks great!

<img width="972" alt="image" src="https://github.com/ndphillips/FFTrees/assets/8480980/3a0b8784-39ac-425d-ad85-fe5707025cce">

**Comment by hneth** [comment]
Thanks for adjusting the repo settings.  I now can see the generated **pkgdown** site, but it doesn't seem to update automatically when changing the package contents, even though some corresponding GitHub actions completed successfully.  (The current main page at <https://ndphillips.github.io/FFTrees/> still is from 2024-05-10.) 

As I used the same setup that works for my other packages, I suspect that this is due to me initializing the corresponding actions in a project fork.  Hence, perhaps you could check or re-initialize the **pkgdown** setup (e.g., using the corresponding **usethis** commands)?
